kratos/python_scripts/gid_output_process.py

```diff
@@ -645,3 +653,48 @@ def __remove_list_files(self):
                         os.remove(f)
                     except WindowsError:
                         pass
+                    
+    #
+    def __remove_post_results_files(self, step_label):
```

Comment:
i think that __ is for builtings, like __init__()

if you want to show something is private it is 1 underscore at the beginning.

still, my proposal would be to accept it for now...

Riccardo